README.md

```diff
@@ -13,12 +13,12 @@ A new pypi release will be automatically triggered by Travis.
 pip install python_moztelemetry
 ```
 
-## Updating histogram_tools.py
-moztelemetry/histogram_tools.py is a mirror of its counterpart from
-[mozilla-central](https://hg.mozilla.org/mozilla-central/raw-file/tip/toolkit/components/telemetry/histogram_tools.py).
+## Updating parse_histograms.py
```

Comment:
Please add a quick note to this section to help future maintainers saying something like:

Note that this file was formerly called `histogram_tools.py` and was renamed in [Bug 1419761](https://bugzilla.mozilla.org/show_bug.cgi?id=1419761).